rpmrc.in

```diff
@@ -80,6 +80,9 @@ optflags: armv6hl -O2 -g -march=armv6 -mfloat-abi=hard -mfpu=vfp
 optflags: armv7l -O2 -g -march=armv7
 optflags: armv7hl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=vfpv3-d16
 optflags: armv7hnl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=neon
+optflags: armv8l -O2 -g -march=armv8-a
+optflags: armv8hl -O2 -g -march=armv8-a -mfloat-abi=hard -mfpu=vfpv4-d16
+optflags: armv8hnl -O2 -g -march=armv8-a -mfloat-abi=hard -mfpu=neon
```

Comment:
There appears to be an armv8 variant of the neon fpu mode, so maybe we should use it? `-mfpu=neon-fp-armv8`